src/Psalm/Internal/Stubs/CoreImmutableClasses.phpstub

```diff
@@ -61,21 +91,33 @@ class Exception implements Throwable
 
     /**
      * @psalm-mutation-free
-     * @return int|string
      */
-    public final function getCode(){}
+    public final function getCode(): int {}
```

Comment:
`getCode()` can return string

Reply:
Are you sure? https://3v4l.org/cJHtA

Reply:
Yes. PDO throw exceptions with string codes.

https://3v4l.org/lP7cb
https://www.php.net/manual/en/exception.getcode.php

Reply:
Indeed. WTF…

Reply:
See fix here: https://github.com/vimeo/psalm/pull/3175